### ISSUE TITLE
Smiles parser does throw error for invalid smiles on google collab

### ISSUE THREAD

**Issue by JanoschMenke** [rdkit/rdkit#5490]
I am using RDKit in Google Collab.

```
!pip install rdkit==2022.3.4
from rdkit.Chem import AllChem as Chem
from rdkit.Chem.Draw import IPythonConsole
```

However, for invalid Smiles `Chem.MolFromSmiles()` does only return `None` rather than throwing an error. If I run the same notebook locally it behaves as t should be and throw an error.

**Comment by wopozka** [comment]
Well, what error are you expecting here? I've just tried in notebook, and None is returned as value if the smiles is wrong.

**Comment by JanoschMenke** [comment]
For example:

```
import rdkit
rdkit.Chem.MolFromSmiles("CCc")
```
>>> [12:36:06] non-ring atom 2 marked aromatic

**Comment by wopozka** [comment]
IMHO it might be collab feature, it just intercepts output and does not print it, but it is only assumption.

**Comment by bp-kelley** [comment]
Could you try the following:

from rdkit import rdBase
rdBase.LogToPythonStderr()

This happens automatically for Jupyter notebooks but may not for collab.

**Comment by ricrogz** [comment]
Hi @JanoschMenke, I think this is expected behavior. Parsing an invalid SMILES should indeed not throw, but return `None`. Apparently, there was a bug that was causing it to throw, which was fixed in #4232.

**Comment by JanoschMenke** [comment]
Hi 

sorry for being inactive @bp-kelley was right. When calling `rdBase.LogToPythonStderr()` it fixed the issue
